test/powershell/Modules/Microsoft.PowerShell.Utility/Import-Csv.Tests.ps1

```diff
@@ -71,3 +19,104 @@ Describe "Import-Csv DRT Unit Tests" -Tags "CI" {
+        $orginalHeader = "Column1","Column2","Column 3"
+        $customHeader = "test1","test2","test3"
+    }
+    # Test set is the same for all file formats
+    foreach ($testCsv in $testCSVfiles) {
+       $FileName = (dir $testCsv).Name
+        Context "Next test file: $FileName" {
+            BeforeAll {
+                if ($FileName -eq "TestImportCsv1.csv") {
+                    # The file does not have header
+                    # (w/o Delimiter here we get throw (bug?)) 
+                    $HeaderParams = @{Header = $orginalHeader; Delimiter = ","}
+                    $CustomHeaderParams = @{Header = $customHeader; Delimiter = ","}
+                } else {
+                    # The files have header
+                    $HeaderParams = @{Delimiter = ","}
+                    $CustomHeaderParams = @{Header = $customHeader; Delimiter = ","}
+                }
+
+            }
+            It "Should be able to call without error" {
+                { Import-Csv $testCsv @HeaderParams } | Should Not Throw
+            }
+
+            It "Should be able to assign to a variable" {
+                $actual = Import-Csv -Path $testCsv @HeaderParams
+
+                $actual                     | Should Not BeNullOrEmpty
+                $actual.GetType().BaseType  | Should Be array
+                $actual.count               | Should Be 4
+            }
+
+            It "Should be able to import all fields" {
+                $actual = Import-Csv -Path $testCsv @HeaderParams
+                $actualfields = $actual[0].psobject.Properties.Name
+                $actualfields | Should Be $orginalHeader
+            }
+
+            It "Should be able to import all fields with custom header" {
+                $actual = Import-Csv -Path $testCsv @CustomHeaderParams
+                $actualfields = $actual[0].psobject.Properties.Name
+                $actualfields | Should Be $customHeader
+            }
+
+            It "Should be able to import rights values" {
+                $actual = Import-Csv -Path $testCsv @HeaderParams
+                $actual[0].'Column1'  | Should Be "data1"
+                $actual[0].'Column2'  | Should Be "1"
+                $actual[0].'Column 3' | Should Be "A"
+            }
+            # Is the test unnecessary? Remove?
+            It "Should be able to use the alias without error" {
+                { ipcsv $testCsv } | Should Not Throw
+            }
+            # Is the test unnecessary? Remove?
+            It "Should have the same output between the alias and the full cmdlet name" {
+                $actualAlias = ipcsv -Path $testCsv @HeaderParams
+                $actualCmdlet = Import-Csv -Path $testCsv @HeaderParams
+                $actualAlias[0].'Column1'  | Should Be $actualCmdlet[0].'Column1'
+                $actualAlias[0].'Column2'  | Should Be $actualCmdlet[0].'Column2' 
+                $actualAlias[0].'Column 3' | Should Be $actualCmdlet[0].'Column 3'
+            }
```

Comment:
yah - remove this test - just validate that the alias is present and is set correctly. There is no real need to actually invoke the alias